packages/gatsby-theme-docs/src/data/top-menu.yaml

```diff
@@ -9,3 +9,17 @@
       href: /../docs-smoke-test
     - label: API Docs Smoke Test
       href: /../api-docs-smoke-test
+
+- menu-title: Another product with long title
```

Comment:
Fake menu items, just to give a taste of how the menu will work when having more than 2 items